### PR TITLE
docs: add to container access guide

### PR DESCRIPTION
**Pull request by refack**

### CODE REVIEW

File: doc/jenkins-guide.md [hunk 1]
```diff
@@ -172,30 +188,48 @@ shutdown -r now
...
+    * Each container should have a matching `systemd` service that starts and stops the
+    container. It's name should be `jenkins-${workername}`, so in this example
+    ```
+    jenkins-test-softlayer-ubuntu1804_container-x64-1
+    ```
+3. To view the logs for a service run:  
+`journalctl -u jenkins-test-softlayer-ubuntu1804_container-x64-1`
+4. To restart a Docker container, restart the associated `systemd` service:
+`systemctl restart jenkins-test-softlayer-ubuntu1804_container-x64-1`
+5. `CONTAINER ID` is needed to console into a Docker container:
```
[the remark this answers is not in the document]
Comment (refack) [reply doc/jenkins-guide.md]: Oy vey, it does look horrible

**Comment by refack** [comment]
@maclover7 updated.

File: doc/jenkins-guide.md [hunk 2]
```diff
@@ -172,30 +188,50 @@ shutdown -r now
...
+1. Figure out the which machine hosts the container. It should be stated in
+the [worker's view][1] on Jenkins
+2. Verify the existence of the container:
+    * To view a list of all active Docker containers, you can run: `docker ps`
+    * To view a list of all active services, you can run:  
+      ```bash
+      systemctl list-units | grep jenkins
+      ```
+    * Each container should have a matching `systemd` service that starts and stops the
+    container. It's name should be `jenkins-${workername}`, so in this example
```
Comment (maclover7) [inline doc/jenkins-guide.md]: it's --> its

**Comment by maclover7** [comment]
After a rebase, this should be good to go

**Comment by sam-github** [comment]
@AshCripps can you see if this info is still relevant? Docs are useful, I hate to lose them in stale PRs if they are still relevant!

**Comment by AshCripps** [comment]
@sam-github Ive rebased the PR and tidied up a few tiny things, want to review before I merge?

**Pull request merged**